FIRO_TSEnsembles/src/main/java/hec/stats/MedianComputable.java

```diff
@@ -0,0 +1,16 @@
+package hec.stats;
+
+import java.util.Arrays;
+
+public class MedianComputable implements Computable {
+    @Override
+    public float compute(float[] values) {
+        //calculate the min of values
```

Comment:
i dont think this comment applies anymore, i would clean it up. leaving it is confusing.

Reply:
removed from code